### PR TITLE
Adding Perf Scenarios for Charts and CRA for Perf Measurement.

### PR DESCRIPTION
**Pull request by ankityadav4**

### CODE REVIEW

File: docs/Charting-Concepts/SankeyChart.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+When the user hovers over nodes, the entire flow associated with that node will be highlighted, encompassing both the links and nodes within the entire flow. Furthermore, the entire flow will be colored to match the node upon which the user has hovered. To enhance visibility, all other nodes will be rendered in white with reduced opacity.Below is an example. Here user has hovered over node with value 62 . 
+![image.png](images/SankeyChartPic5.png)
+
+Hovering on the node with height less than 24px will display a callout with details of node name and weight. 
+#### Hover Over links
+When the user hovers over a link, the entire flow will be highlighted. All nodes will retain their original colors, while the links themselves will adopt a gradient color derived from the source and target nodes. Additionally, a callout will appear, displaying details about the link, such as the source node's name, weight, and the target node's name. To enhance the visual appeal, the link border will also transition into a gradient when the link is hovered over. Below is an example of a link being hovered over, in this case, between node 4 and node 12.
+
+![image.png](images/SankeyChartPic6.png)
```
Comment (AtishayMsft) [inline docs/Charting-Concepts/SankeyChart.md:105]: put the images in docs/assets/images folder

File: tools/Performance/CRA/my-app/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `npm run eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+
+If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+
```
Comment (AtishayMsft) [inline tools/Performance/CRA/my-app/README.md:39]: Could you add few steps how to measure performance for different scenarios using this test app

File: docs/Charting-Concepts/AreaChart.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
+# Contributor guide: Area Chart
+
+![Areachartpic1.png](images/Areachartpic1.png)
```
Comment (AtishayMsft) [inline docs/Charting-Concepts/AreaChart.md:3]: Could you add a section in https://github.com/microsoft/fluentui-charting-contrib/blob/main/apps/docsite/sidebars.ts

{
      type:'category',
      label:'Charting Concepts',
      items:[
        {
          type: 'autogenerated',
          dirName:'Charting-Concepts',
        }
      ]
    },

Comment (ankityadav4) [reply docs/Charting-Concepts/AreaChart.md]: Done

File: tools/Performance/CRA/my-app/README.md (hunk 2 above)
Comment (ankityadav4) [reply tools/Performance/CRA/my-app/README.md:39]: I have already created the Read me in the Perf folder where i have mentioned all the steps.

File: docs/Charting-Concepts/SankeyChart.md (hunk 1 above)
Comment (ankityadav4) [reply docs/Charting-Concepts/SankeyChart.md]: Done

File: tools/Performance/CRA/my-app/README.md (hunk 2 above)
Comment (AtishayMsft) [reply tools/Performance/CRA/my-app/README.md:39]: Have a single readme file

**Review by AtishayMsft — APPROVED** [review]

**Comment by AtishayMsft** [comment]
Few other relevant PRs
https://github.com/microsoft/fluentui/pull/26743
https://github.com/microsoft/fluentui/pull/28049

**Pull request merged**